llmriddles/questions/level5.py

```diff
@@ -44,10 +44,14 @@ def _checker_1(question_text: str, user_text: str, answer_text: str, lang: str):
     answer_text = answer_text.strip()
     user_text = user_text.strip()
 
+    if lang == 'en':
+        answer_text = answer_text.split()
+        user_text = user_text.split()
+
     if count_words(user_text) < 10:
```

Comment:
Here, count_words receives a string as input. It will check the length of string under both languages. So you can split the english string into words after this IF

Reply:
I see. count_words always receive the raw input and treat language specifications internally.  Thanks for pointing it out.